packages/cfd/src/Containers/jurisdiction-modal/__test__/jurisdiction-card-back.spec.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import JurisdictionCardBack from '../jurisdiction-card-back';
+
+describe('<JurisdictionCardBack />', () => {
+    type TMockProps = {
+        card_classname: string;
+        toggleCardFlip: jest.Mock;
+        verification_docs: ('document_number' | 'name_and_address' | 'selfie' | 'identity_document')[];
+    };
+
+    const mock_props: TMockProps = {
+        card_classname: 'test_classname',
+        toggleCardFlip: jest.fn(),
+        verification_docs: [],
+    };
+
+    it('should render JurisdictionCardBack without any required submission if verification_docs is empty', () => {
+        render(<JurisdictionCardBack {...mock_props} />);
+        const container = screen.getByTestId('dt_jurisdiction_card_back');
+        expect(container).toHaveClass(
+            'test_classname__card-content-container',
+            'test_classname__card-flipped-container'
+        );
+        expect(screen.getByText('We need you to submit these in order to get this account:')).toBeInTheDocument();
+        expect(screen.queryByText('A selfie of yourself.')).not.toBeInTheDocument();
+        expect(screen.queryByText('Document number (identity card, passport)')).not.toBeInTheDocument();
+        expect(
+            screen.queryByText(
+                'A recent utility bill (electricity, water or gas) or recent bank statement or government-issued letter with your name and address.'
+            )
+        ).not.toBeInTheDocument();
+        expect(
+            screen.queryByText('A copy of your identity document (identity card, passport)')
+        ).not.toBeInTheDocument();
+        expect(screen.getByText('Your document is pending for verification.')).toBeInTheDocument();
```

Comment:
line 36-39,  are common in all test cases, can we write a common test function for this?